homeassistant/components/schedule/strings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "title": "Schedule",
+  "state": {
+    "_": {
+      "off": "[%key:common::state::off%]",
+      "on": "[%key:common::state::on%]"
```

Comment:
Alternatively, it could be translated as "Active" & "Inactive"

Reply:
Active/Inactive would be the most correct. Definitely will be confusing when setting up triggers. Should we just make the actual state `active`/`inactive` ?